include/compose.bash

```diff
@@ -261,6 +261,15 @@ logspout:
     entrypoint: ["/bin/sh"]
     command: -c 'sleep 1; ROUTE_URIS=\$\$LOGSINK_PORT_3333_TCP /bin/logspout'
 
+mail:
+    ports:
+        - "$PRIVATE_IP:25:25"
+    environment:
+        - maildomain=mail.hortonworks.com
+        - smtp_user=admin:cloudbreak
```

Comment:
This looks a username password pair for me. Maybe environment variables would be better .